### ISSUE TITLE
[BUG]  Invalid Tailwind CSS classnames order  tailwindcss/classnames-order

### ISSUE THREAD

**Issue by productdevbook** [francoismassart/eslint-plugin-tailwindcss#105]
```
class="flex justify-center items-center w-9 h-9 rounded-md hover:bg-toprak-300 dark:hover:bg-toprak-600"
```

vscode fixed but terminal yarn lint 
  149:21  warning  Invalid Tailwind CSS classnames order  tailwindcss/classnames-order

**Comment by francoismassart** [comment]
Please set up a repo demonstrating the issue because I can't reproduce this.

I added a valid test case on my local 3.3.0

```js
    {
      code: `<div class="flex justify-center items-center w-9 h-9 rounded-md hover:bg-toprak-300 dark:hover:bg-toprak-600">issue #105</div>`,
    },
```

Which passes

**Comment by francoismassart** [comment]
Please provide a demo repo because I cannot reproduce the issue. Re-open the issue when you can provide a demo project.
